src/cinder/audio/OutputNode.cpp

```diff
@@ -96,7 +98,21 @@ OutputDeviceNode::OutputDeviceNode( const DeviceRef &device, const Format &forma
 	mDidChangeConn = mDevice->getSignalParamsDidChange().connect( bind( &OutputDeviceNode::deviceParamsDidChange, this ) );
 
 	mInterruptionBeganConn = Context::deviceManager()->getSignalInterruptionBegan().connect( [this] { disable(); } );
-	mInterruptionEndedConn = Context::deviceManager()->getSignalInterruptionEnded().connect( [this]( bool shouldResume ) { enable(); } );
+	mInterruptionEndedConn = Context::deviceManager()->getSignalInterruptionEnded().connect( [this]( bool resumeImmediately ) {
```

Comment:
I was imagining that the `DeviceManagerAudioSession` would handle this logic internally, and there would be no need to pass the `resumeImmediately` flag at all. This app signal is really only needed on the iOS platform at current.

Also it seems a bit simpler to me if `DeviceManagerAudioSession` always registered for the `signalDidBecomeActive` and then when that gets called, it checks some sort of flag like `mIsCurrentlyInterrupted` or something. If it is, then it will call the `getSignalInterruptionEnded().emit()` upon app becoming active again.